index.html

```diff
@@ -80,11 +66,178 @@ <h1 class="primary-title font-family-poppins">I'm Belden.<br> Glad to see you!</
+
+            <ul class="column-links">
+               <li class="column-link"><a href="#">html</a></li>
+               <li class="column-link"><a href="#">css</a></li>
+               <li class="column-link"><a href="#">javascript</a></li>
+            </ul>
+
+
+            <a href="" class="column-button"> see project</a>
+         </div>
+
+         <div class="col-1 column-item">
+            <img src="images\Snapshoot (1).jpg" alt="">
+            <h2 class="col-title">Multi-post <br> stories</h2>
+
+            <ul class="canopy">
+               <li class="canopy-title ">canopy</li>
+               <li class="can-items">
+                 <span>• </span>  backend-dev <span>• </span>
+               </li>
+               <li class="can-items">2015</li>
+            </ul>
+
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+            <ul class="column-links">
+               <li class="column-link"><a href="#">html</a></li>
+               <li class="column-link"><a href="#">css</a></li>
+               <li class="column-link"><a href="#">javascript</a></li>
+            </ul>
+
+
+            <a href="" class="column-button">see project</a>
+         </div>
+
+           
+        </section>
+
+        <section class="about-me font-family-poppins">
+           
+            <div class="about-details">
+                <h2 class="about-title">About <br> Myself</h2>
+                <p>
+                    Hello I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+                </p>
+                <a href="#" class="social-header font-family-poppins">LET'S CONNECT</a>
+                <ul class="social-media">
+                    <li>
+                        <a href="#" target="_blank" rel="noopener noreferrer">
+                            <i class="fa-brands fa-square-github"><span class="sr-only">github</span></i>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" target="_blank" rel="noopener noreferrer">
+                            <i class="fa-brands fa-twitter"><span class="sr-only">twitter</span></i>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" target="_blank" rel="noopener noreferrer">
+                            <i class="fa-brands fa-linkedin-in"><span class="sr-only">linkedin</span></i>
+                        </a>
+                    </li>
+                    <li>   
+                        <a href="#" target="_blank" rel="noopener noreferrer">
+                        <i class="fa-brands fa-whatsapp"><span class="sr-only">whatsapp</span></i>
+                        </a>
+                    </li>                    
+                </ul>
```

Comment:
- [ ] As mentioned before as part of the Figma design you have to give `width: 20px` to the social media icons using `font-size` to match as much as possible with the template. It means making it bigger :+1: 

**Your design**

![Screenshot from 2023-01-14 08-28-13](https://user-images.githubusercontent.com/105117832/212474209-ae5370aa-58f1-469f-bae6-4536e21a241b.png)

**Figma design**

![Screenshot from 2023-01-14 08-28-51](https://user-images.githubusercontent.com/105117832/212474241-fbaa3094-2da6-4a49-adfb-056f307f8cfe.png)